src/pages/Episode/index.tsx

```diff
@@ -1,6 +1,84 @@
-import React from "react"
-import "./styles.scss"
+import React, { useEffect } from "react";
+import { useQuery } from "@apollo/client";
+
+import EpisodeCard from "../../components/EpisodeCard";
+
+import { GetEpisodes } from "../../queries/queries";
+import { IEpisode } from "../../types/interfaces";
+import ShowCount from "../../components/ShowCount";
 
 export default function Episode() {
-  return <div>Episode</div>
+  const [episodes, setEpisodes] = React.useState<IEpisode[]>([]);
+  const [episodeArray, setEpisodeArray] = React.useState<JSX.Element[]>([]);
+  const { loading, error, data, refetch } = useQuery(GetEpisodes, {
+    variables: {
+      page: 1
+    }
+  });
+
+  useEffect(() => {
+    if (loading === false && data) {
```

Comment:
!loading

Reply:
The documentation says that the loading variable can be null so I ended up using === false to make sure everything goes as expected